src/app/app.component.ts

```diff
@@ -32,10 +35,19 @@ export class AppComponent implements OnInit, OnDestroy {
         filter(
           (status: InteractionStatus) => status === InteractionStatus.None,
         ),
+        first(),
+        switchMap(() => this.store.select(selectProfile)),
+        filter((profile) => !!profile),
+        switchMap((profile) =>
+          this.userService.initUser(
```

Comment:
Wird der User mit "Unknown" erstellt, wenn die Attribute userPrincipalName und displayName nicht gesetzt sind?

Reply:
Angepasst dass die Daten jetzt aus dem JWT Token gelesen werden.

Reply:
Wenn wir diesen MR mergen bevor es den entsprechenden BE-Endpunkt gibt, ist die App unbrauchbar, richtig?

Reply:
Ja :)